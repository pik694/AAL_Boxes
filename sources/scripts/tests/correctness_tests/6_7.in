6
73 76 48
88 47 48
28 53 80
37 15 57
54 4 90
42 92 6

