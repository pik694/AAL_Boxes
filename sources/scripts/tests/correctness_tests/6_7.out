4 5 3 2 1 0 
657811->626176
